src/manifest.ts

```diff
@@ -69,6 +69,10 @@ export class SlackManifest {
   }
 
   private registerFeatures() {
+    this.definition.workflows?.forEach((workflow) => {
+      workflow.registerStepFunctions(this);
+      workflow.registerParameterTypes(this);
+    });
```

Comment:
I noticed that running `hermes manifest` will register the functions I use in the workflow but the workflow itself isn't being added. I think this is because we seem to be missing the logic in the manifest export method to take the workflows from the manifest definition and add it to the manifest we return. Should look something like
```ts
if (def.workflows) {
      manifest.workflows = def.workflows?.reduce((acc = {}, fn) => {
        acc[fn.id] = fn.export();
        return acc;
      }, {} as ManifestSchema["workflows"]);
    }
```